.github/workflows/publish.yml

```diff
@@ -0,0 +1,30 @@
+
+name: Node.js Package
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2.4.1
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+          scope: '@pocket-tools'
+      - run: npm ci
+      - run: npm run build
+      - run: npm run test
```

Comment:
the build action is failing because jest does not exist but not blocking since you are yet to add all the library-specific code and tests